LogAnalysis/HealthAnalysisModule/GetClusterGroup.psm1

```diff
@@ -0,0 +1,29 @@
+<#
```

Comment:
Change the folder name here, the better known name of this log folder is SDDC, so better change the folder name to SddcAnalysisModules